app/assets/javascripts/dradis/plugins/csv/upload.js

```diff
@@ -49,7 +64,72 @@ document.addEventListener('turbolinks:load', function() {
       return valid;
     });
 
-    function validateNodeSelected() {
+    // Private methods
+
+    function _setDradisFieldSelect($select) {
+      var rtpFields = $('[data-behavior=dradis-datatable]').data('rtp-fields');
+      if (rtpFields) {
+        var fields = rtpFields[$select.val()],
+            $fieldSelect = $select.closest('tr').find('[data-behavior=field-select]');
+
+        if ($select.val() in rtpFields){
+          if (fields.length > 0) {
+            $fieldSelect.empty();
+            fields.forEach(function(value) {
+              $fieldSelect
+                .removeAttr('disabled')
```

Comment:
SInce we are not disabling the select element itself anymore, we don't need this line anymore since we'll never need to re-enable a select.

```suggestion
```